src/bundle/Resources/views/themes/admin/ui/menu/user_menu.html.twig

```diff
@@ -19,3 +19,17 @@
     {% endif %}
     {{ parent() }}
 {% endblock %}
+
+{% block linkElement %}
+    {% import _self as knp_menu %}
+    <a href="{{ item.uri }}"{{ knp_menu.attributes(item.linkAttributes) }}>
+        {% if item.getExtra('icon') is not null %}
+            <svg class="ez-icon ez-icon--small-medium ez-user-menu__item-icon" aria-hidden="true">
+                <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ item.getExtra('icon') }}"></use>
```

Comment:
People may want to add custom icon to the list for custom items. I think we should store the whole path (`bundles/ezplatformadminui/img/ez-icons.svg#bookmark-manager`) in the `icon` not only the identifier

Reply:
Good idea!